exercises/J-array-get-set/exercises2.js

```diff
@@ -7,6 +7,7 @@
 */
 
 var numbers = [1, 2, 3]; // Don't change this array literal declaration
+numbers[3] = 4;
```

Comment:
Hey Yunus :wave: 
What if we had to multiple elements to the end of the array? Would we have to pass the index each time?
Is there another way to insert an element at the end of the array?